deployments/cloud-managed/.env.example

```
@@ -21,13 +21,15 @@ WITNESS_DATA_RESIDENCY="Operator-managed host (Your Institution)"
 
 # ── Hostnames — the six lines a domain cutover changes ─────────────────────
-WITNESS_WEB_HOST=witness.example.org
-WITNESS_OIDC_HOST=witness-id.example.org
-NEXT_PUBLIC_WITNESS_API_URL=https://witness-api.example.org
-NEXT_PUBLIC_WITNESS_BASE_URL=https://witness-api.example.org
-WITNESS_WEB_ORIGIN=https://witness.example.org
-WITNESS_OIDC_REDIRECT_URI=https://witness-api.example.org/api/v1/auth/callback
-WITNESS_WEB_BASE_URL=https://witness.example.org
+WITNESS_PUBLIC_URL=https://buildwithwitness.com
+WITNESS_WEB_HOST=app.buildwithwitness.com
+WITNESS_API_HOST=api.buildwithwitness.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WITNESS_API_HOST key should go before the WITNESS_PUBLIC_URL key

(UnorderedKey)